src/ResourceManager/Insights/Commands.Insights/Diagnostics/SetAzureRmDiagnosticSettingCommand.cs

```diff
@@ -34,6 +33,7 @@ public class SetAzureRmDiagnosticSettingCommand : ManagementCmdletBase
     {
         public const string StorageAccountIdParamName = "StorageAccountId";
         public const string ServiceBusRuleIdParamName = "ServiceBusRuleId";
+        public const string EventHubRuleIdParamName = "EventHubAuthorizationRuleId";
```

Comment:
@gucalder You can actually tell this by using MyInvocation.BoundParemeters, rather than a parameter set.